src/controller/CHIPDeviceControllerFactory.h

```diff
@@ -170,6 +170,10 @@ struct FactoryInitParams
     chip::app::DataModel::Provider * dataModelProvider = nullptr;
 
     std::optional<Inet::InterfaceId> interfaceId;
+
+    // The minimum backoff interval for LIT devices. This is used to calculate
+    // the sigma1 retransmission timeout for LIT devices.
```

Comment:
This does not adequately describe the spec-compliance implications of setting this value.

Reply:
The updates do not address my comment.  Again, see https://github.com/project-chip/connectedhomeip/pull/41747#discussion_r2487610392

Reply:
Just put // Note: This parameter is not spec-compliant. thanks